pages/osx/lp.md

```diff
@@ -0,0 +1,15 @@
+# lp
+
+> print files
+
+- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
+
+'lp -d {{device_name}} {{path/to/filename}}
```

Comment:
Gotcha.  The files are edited.  I just have to remove that line from the script and I should be good.
> On Nov 6, 2015, at 9:16 PM, Waldir Pimenta <notifications@github.com> wrote:
> 
> In pages/osx/lp.md <https://github.com/tldr-pages/tldr/pull/318#discussion_r44206079>:
> 
> > @@ -0,0 +1,15 @@
> > +# lp
> > +
> > +> print files
> > +
> > +- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
> > +
> > +'lp -d {{device_name}} {{path/to/filename}}
> Again, that is probably an OSX-specific feature that doesn't translate to other systems. Besides, I don't think this would be confusing if instead of instructions to look at the printers using the GUI, we suggest using lpstat.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tldr-pages/tldr/pull/318/files#r44206079>.
>